### ISSUE TITLE
Does not work with Angular CLI 13.0.1

### ISSUE THREAD

**Issue by nagkumar** [valor-software/ng2-bootstrap#6353]
```
npm ERR! peer dep missing: @angular/animations@^11.0.0 || ^12.0.0, required by ngx-bootstrap@7.1.2
npm ERR! peer dep missing: @angular/common@^11.0.0 || ^12.0.0, required by ngx-bootstrap@7.1.2
npm ERR! peer dep missing: @angular/core@^11.0.0 || ^12.0.0, required by ngx-bootstrap@7.1.2
npm ERR! peer dep missing: @angular/forms@^11.0.0 || ^12.0.0, required by ngx-bootstrap@7.1.2
```
Pl. upgrade

**Comment by valorkin** [comment]
will be supported in v8

**Comment by nagkumar** [comment]
Any time lines for v8 release pl.

**Comment by sebasdeweert** [comment]
When will v8 be released? Is there any CLI 13.0.1 supported branch I can use in the meantime?

**Comment by nagkumar** [comment]
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-bootstrap@7.1.2
npm WARN Found: @angular/animations@13.0.0
npm WARN node_modules/@angular/animations
npm WARN   @angular/animations@"~13.0.0" from the root project
npm WARN   1 more (@angular/platform-browser)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/animations@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN node_modules/ngx-bootstrap
npm WARN   ngx-bootstrap@"^7.1.2" from the root project
npm WARN
npm WARN Conflicting peer dependency: @angular/animations@12.2.13
npm WARN node_modules/@angular/animations
npm WARN   peer @angular/animations@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN   node_modules/ngx-bootstrap
npm WARN     ngx-bootstrap@"^7.1.2" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-bootstrap@7.1.2
npm WARN Found: @angular/common@13.0.0
npm WARN node_modules/@angular/common
npm WARN   @angular/common@"~13.0.0" from the root project
npm WARN   4 more (@angular/forms, @angular/platform-browser, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/common@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN node_modules/ngx-bootstrap
npm WARN   ngx-bootstrap@"^7.1.2" from the root project
npm WARN
npm WARN Conflicting peer dependency: @angular/common@12.2.13
npm WARN node_modules/@angular/common
npm WARN   peer @angular/common@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN   node_modules/ngx-bootstrap
npm WARN     ngx-bootstrap@"^7.1.2" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-bootstrap@7.1.2
npm WARN Found: @angular/core@13.0.0
npm WARN node_modules/@angular/core
npm WARN   @angular/core@"~13.0.0" from the root project
npm WARN   6 more (@angular/animations, @angular/common, @angular/forms, ...)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/core@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN node_modules/ngx-bootstrap
npm WARN   ngx-bootstrap@"^7.1.2" from the root project
npm WARN
npm WARN Conflicting peer dependency: @angular/core@12.2.13
npm WARN node_modules/@angular/core
npm WARN   peer @angular/core@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN   node_modules/ngx-bootstrap
npm WARN     ngx-bootstrap@"^7.1.2" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: ngx-bootstrap@7.1.2
npm WARN Found: @angular/forms@13.0.0
npm WARN node_modules/@angular/forms
npm WARN   @angular/forms@"~13.0.0" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @angular/forms@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN node_modules/ngx-bootstrap
npm WARN   ngx-bootstrap@"^7.1.2" from the root project
npm WARN
npm WARN Conflicting peer dependency: @angular/forms@12.2.13
npm WARN node_modules/@angular/forms
npm WARN   peer @angular/forms@"^11.0.0 || ^12.0.0" from ngx-bootstrap@7.1.2
npm WARN   node_modules/ngx-bootstrap
npm WARN     ngx-bootstrap@"^7.1.2" from the root project

```
Wish this issue is kept open and not closed unless v8 is released, as this bug does exist in "ngx-bootstrap": "^7.1.2",
which is the latest release.
